tests/state_test_util.go

```diff
@@ -115,6 +118,12 @@ type stTransactionMarshaling struct {
 	PrivateKey hexutil.Bytes
 }
 
+var isTestExecutionSpecState bool
```

Comment:
Is this correct?
If `isTestExecutionSpecState = false`, test runs `kaiachain/kaia-core-tests`
If `isTestExecutionSpecState = true`, test runs ethereum tests.

Reply:
Yes. Only if `isTestExecutionSpecState = true`, simulating Ethereum stuff is run.

Reply:
Maybe we should call it something like `isEthSpecCompatibleMode` or something.. cc @hyunsooda

Reply:
Is your proposal about a variable name?